lib/mongo_storage.ex

```diff
@@ -290,7 +290,7 @@ defmodule ApplicationRunner.MongoStorage do
       [_, hex_id] ->
         case BSON.ObjectId.decode(hex_id) do
           :error ->
-            BusinessError.not_an_object_id_tuple()
+            {:error, BusinessError.not_an_object_id_tuple()}
```

Comment:
```suggestion
            BusinessError.not_an_object_id_tuple()
```

Why do you add `:error` ? It is already present in the call to `BusinessError` thanks to `_tuple()`.

This looks clean like this but if you expand the call to BusinessError to its real value it looks odd.
{:error, {:error, ...}}